components/admin_console/data_grid/data_grid_search.tsx

```diff
@@ -78,6 +78,7 @@ class DataGridSearch extends React.PureComponent<Props, State> {
                         placeholder={Utils.localizeMessage('search_bar.search', 'Search')}
                         onChange={this.handleSearch}
                         value={this.props.term}
+                        data-testid='search-input'
```

Comment:
Please change to camelCase for consistency with our docs - https://developers.mattermost.com/contribute/webapp/end-to-end-tests/#writing-specs. Apply the same to other instance.

Reply:
Ok! Done. Thanks ;)